From 0e50f4aa466679d2a7ecb12b9dff858ed52bbfae Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 25 Jan 2024 12:12:08 +0200
Subject: [PATCH] Revert "[SW-165644] make: use new defconfig with stack size"

This reverts commit 9bf0e347e0ebe11b7aa54a8527bf15c4db951261.

revert patch as it is causing to reduce stack size for all
zephyr. Until this is changed zephyr application specific,
revert this change.

Change-Id: I3aaed658652e3e9bb88872debb0cf1863a98eb18
---

diff --git a/configs/zephyr_arc_stack_defconfig b/configs/zephyr_arc_stack_defconfig
deleted file mode 100644
index 295e6b5..0000000
--- a/configs/zephyr_arc_stack_defconfig
+++ /dev/null
@@ -1,7 +0,0 @@
-CONFIG_MAIN_STACK_SIZE=1384
-CONFIG_IDLE_STACK_SIZE=256
-CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
-CONFIG_ARC_EXCEPTION_STACK_SIZE=512
-CONFIG_ISR_STACK_SIZE=1792
-CONFIG_IPM_CONSOLE_HL_STACK_SIZE=384
-CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=512
diff --git a/package/zephyr/zephyr.mk b/package/zephyr/zephyr.mk
index 620b0b3..8743699 100644
--- a/package/zephyr/zephyr.mk
+++ b/package/zephyr/zephyr.mk
@@ -18,12 +18,6 @@
 
 ZEPHYR_CONF_DEV_EXTRA_CFLAGS += -DCONFIG_HL_STACK_OVERFLOW_REDUCE_SIZE=$(STACK_REDUCTION_SIZE)
 
-ifdef BR2_arcle
-ifneq ($(STACK_REDUCTION_SIZE),0)
-ZEPHYR_CONF_OPTS += -DOVERLAY_CONFIG=$(BR2_EXTERNAL_HABANALABS_PATH)/configs/zephyr_arc_stack_defconfig
-endif
-endif
-
 DEVICE_NAME = $(firstword $(subst _, ,$(HL_DEV)))
 
 # zephyr uses west tool to perform repo management, installation
